Dockerfile

```diff
@@ -9,7 +9,8 @@
 # either express or implied. See the License for the specific language governing permissions
 # and limitations under the License.
 # ----------------------------------------------------------------------------------------
-FROM eclipse-temurin:11-alpine as builder
+FROM azul/zulu-openjdk-alpine:17 as builder
+RUN apk add --no-cache binutils
```

Comment:
Out of curiosity, why is `binutils` needed?

Reply:
`--strip-debug` requires `objcopy` starting with JDK 13 - https://github.com/docker-library/openjdk/issues/351.

We can either add `binutils` in this builder layer and continue using `--strip-debug`, or switch to `--strip-java-debug-attributes`, which nowadays does what `--strip-debug` used to do in JDK 11. See https://bugs.openjdk.org/browse/JDK-8219207 for more details.